views/ppe-thanks.ejs

```diff
@@ -1,15 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <%- include('partials/head') %>
-  </head>
+</head>
+
 <body class=" bg-gradient">
 
     <%- include('partials/navbar') %>
     <div class="container mt-5">
         <div class="jumbotron">
             <h1 class="display-4">Thank you!</h1>
-            <p class="lead">Subscribe to get notified when we find a match</p>
+            <p>We have sent you an email confirmation along with a link to view the status of your application.
+                Once we match someone with you, we will send you an email notification or contact you directly
+                in case of urgency.
+            </p>
+            <p class="lead">If you wish to receive notifications in addition to emails, please click the Subscribe button below.</p>
```

Comment:
On L28 there's an unnecessary GET of `/config.js`

Reply:
No it's being used by the `subscribeUserToPush` function on L96